src/de/grobox/blitzmail/send/MailSender.java

```diff
@@ -108,7 +108,7 @@ private MimeBodyPart getMimeBodyPart(JSONObject attachment) throws IOException,
 	private MimeMessage getMessage() throws Exception {
 		MimeMessage message = new MimeMessage(session);
 		message.setFrom(); // uses mail.user property
-		message.setSentDate(new Date());
```

Comment:
Does `Date(long time)` take `null` as an argument?

Reply:
You probably want to use `MAIL_DATE` instead of `"date"`.

Reply:
It seems not. It should indeed not be null since we always put it into the jsonobj. But if it is null, should we just do `new Date()`?

Reply:
yes, in case people had mails stored (without the date), update and then send the mail.

Reply:
Check!